.changes/next-release/feature-55ae3b21-a7ab-4f46-8165-10b533c54634.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Add support for creating and running Lambdas with dotnet core 3.1. The ability to debug will come in a future release"
```

Comment:
nit: "The ability to debug" -> "Debug support"

Reply:
Fixed